src/main/resources/assets/carpet/scripts/distance_rc_1.sc

```diff
@@ -51,5 +64,15 @@ calculate(end_pos)->(
         'w  - Spherical: ', 'w '+spherical+'\n',
         'w  - Cylindrical: ', 'w '+cylindrical+'\n',
         'w  - Manhattan: ', 'w '+manhattan
```

Comment:
Parity: In old command, distance numbers are in bold. Not sure if making them bold or what, but at least something to make it different.

Reply:
ye I hadnt read old code properli